pkgs/top-level/all-packages.nix

```diff
@@ -12075,6 +12075,8 @@ let
 
   mutt-kz = callPackage ../applications/networking/mailreaders/mutt-kz { };
 
+  openshift = callPackage ../applications/networking/cluster/openshift { };
```

Comment:
Name mismatch: the package is called `openshift` here but `openshift-origin` elsewehere.